### ISSUE TITLE
Editing the main page title does not update the entry title Place Name

### ISSUE THREAD

**Issue by cartosattva** [cestastanford/landtalk#19]
It looks like these are two different fields in the database. Is there a good reason for this? I noticed this when editing page titles to correct spellings. Seems like we should make them the same field.

**Comment by codyml** [comment]
Yep, this is an awkward side-effect of using ACF to generate the form instead of handwriting it:
 - The Place Name field is what’s rendered on the front-end both in the conversation excerpts and the conversation single page.
 - ACF forms can only update ACF fields, not the post title, so originally the form was just creating conversations named “Conversation 14.”
 - I added some code so when you submit a conversation, it sets the post title to whatever the Place Name is, so you could identify conversations in the list in the admin panel more easily.  But, if either or both are edited later they’ll get out of sync, and only the Place Name will be used for rendering.

I think we can do some WP magic to fix this:
 - Hide the post title box from the WP admin edit screen with ACF
 - Set the Place Name field to update the post title whenever it’s updated, so when viewed as a list in WP admin, the title will be the Place Name.  [this](https://www.advancedcustomfields.com/resources/acf-update_value/) ACF hook should work.
